_pages/opportunity-grants.md

```diff
@@ -58,7 +58,7 @@ You can apply for 3 types of grant:
 - Visa application fee: reimbursement for the cost of a UK Standard Visitor visa (up to £100).
 
 {% comment %}
```

Comment:
Double checking whether it’s intentional this is still commented out?

Reply:
Yes, that's correct, I don't want to open the grant application until the form is reviewed by everyone and we're happy about the info we're collecting